code/game/gamemodes/shadowling/thrall_abilities.dm

```diff
@@ -0,0 +1,121 @@
+
+/obj/effect/proc_holder/spell/targeted/sling/lesser_glare //a defensive ability, nothing else. can't be used to stun people, steal tasers, etc. Just good for escaping
+	name = "Lesser Glare"
+	desc = "Makes a single target dizzy for a bit."
```

Comment:
Because of the snowballing nature of shadowlings, removing this ability's stunning capability still doesn't fix it. Taunting Tirade was removed from CC because it was a guaranteed escape, and Tirade had drawbacks and could be stopped with a disarm. This one doesn't and can't. 25 confusedness is a massive amount, and likely enough to keep the target confused well after the ability is ready again.